specifyweb/frontend/js_src/lib/components/Statistics/Categories.tsx

```diff
@@ -125,7 +125,7 @@ export function Categories({
     | ((
         categoryIndex: number,
         itemIndex: number,
-        newLabel: LocalizedString
+        newLabel: string
```

Comment:
why was this change made? `LocalizedString` should be used instead of `string` for every string that is either localized by us or comes from the user (thus presumed to be localized)

by using `LocalizedString` type, TypeScript can catch cases when we forgot to localize a string

same applies for all the other cases where you replaced `LocalizedString` with `string`